src/qililab/platform/platform.py

```
@@ -1367,3 +1427,18 @@ def compile(
         )
 
         return compiled_programs, final_layout
+
+    def draw_oscilloscope_platform(self, qprogram: QProgram, averages_displayed: bool = False):
+        """Draw the QProgram using QBlox Compiler
```

Pulse plotting only works for Qblox for now. what happens if the user sends a QuantumMachines QProgram here?

---

done, now if a quantum machine runcard is provided it raises an error instead of a warning